js/index.js

```diff
@@ -20,11 +34,13 @@ webix.ready(function(){
             {view: "label", label:"My App"},
             {
                view:"button",
+               id:"btn_profile",
```

Comment:
normally we do not define ids for components if we do not need to access them later

Reply:
Was edited